app/lib/getNearbyServices.js

```diff
@@ -1,23 +1,43 @@
-const log = require('../../app/lib/logger');
+const log = require('../lib/logger');
 const esClient = require('./esClient');
-const filterServices = require('./filterServices');
+const addMessages = require('./addMessages');
+const getDateTime = require('./getDateTime');
 const VError = require('verror').VError;
 const esGetPharmacyHistogram = require('./promHistograms').esGetPharmacy;
+const addMessagesHistogram = require('../lib/promHistograms').addMessages;
 
 async function getNearbyServices(searchCoordinates, limits) {
   let endTimer;
-  let pharmacies;
+  let nearbyPharmacies;
+  let openPharmacies;
 
   try {
     endTimer = esGetPharmacyHistogram.startTimer();
-    pharmacies = await esClient.getPharmacies(searchCoordinates, limits.searchRadius);
+    openPharmacies = await esClient.getOpenPharmacies(
+      getDateTime(),
+      searchCoordinates,
+      limits.searchRadius,
+      limits.open
+    );
+    nearbyPharmacies = await esClient.getPharmacies(
+      searchCoordinates,
+      limits.searchRadius,
+      limits.nearby
+    );
   } catch (err) {
     log.error({ err: new VError(err) });
     throw err;
   } finally {
     endTimer();
   }
-  return filterServices(pharmacies, limits);
+
+  const fiterTimer = addMessagesHistogram.startTimer();
```

Comment:
What is the reasoning for adding the timer for the adding of messages?

Reply:
it was to keep the same structure as before with metrics for the ES searches, then the client side processing. As the client side processing is now greatly reduced, I not sure there is much value in it.